database/schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "collections": [
+      {
+        "name": "Activities",
+        "fields": [
+          { "name": "activityId", "type": "Document ID" },
+          { "name": "description", "type": "String" },
+          { "name": "tickets", "type": "Number" }
```

Comment:
i think this was said we were going to do an activity for 1 but this would give flexibility to increment up if we want to later on